qutebrowser/browser/browsertab.py

```diff
@@ -196,6 +196,10 @@ def next_result(self, *, result_cb=None):
         """
         raise NotImplementedError
 
+    def searching(self):
+        """Return True if we are currently searching or not."""
+        raise NotImplementedError
+
```

Comment:
Hmm, I guess we probably need two status variables then, does `search_displayed` sound ok? I think that would remove the ambiguity.